### ISSUE TITLE
Installation or Repair Failed - Mac OS X EL Capitan beta 5

### ISSUE THREAD

**Issue by gaivoronsky** [Tunnelblick/Tunnelblick#264]
Hi,
When I am trying install Tunnelblick 3.6beta08 build 4371.
I was having the "Installation or Repair Failed" issue. After several uninstall/reinstall cycles, I repaired my boot disk permissions. But still getting same error. Here is console log https://gist.github.com/gaivoronsky/ca2995cc439631a71c60

**Comment by jkbullard** [comment]
This doesn't happen to me with a fresh copy of beta 5.

Try [uninstalling Tunnelblick completely](https://tunnelblick.net/cUninstall.html), then installing a fresh copy. If that fails, please provide all log entries from the uninstall through the failed install. That should include log entries that show *why* the install was being done, which could point to what exactly is failing. (What you posted seems to be the log from an install attempt after a first, failed attempt, so it doesn't show the original failure or why the later attempt is being made.)

**Comment by gaivoronsky** [comment]
I was following your suggestion but still failing on installation step and have same error.
Installation process attempt many times but not success, in console log I see "Timed out waiting for /tmp/tunnelblick-authorized-running"

Activity Monitor shows several running process "installer"

![screenshot 2015-07-31 16 31 28](https://cloud.githubusercontent.com/assets/254732/9008259/ab3cf472-37a1-11e5-8a01-0d0af40cdccc.png)

Here is console output https://gist.github.com/gaivoronsky/d87d6afb2857cca83faf 

/tmp/tunnelblick-installer-log.txt is empty
/Library/Application Support/Tunnelblick/Logs don't contain any log files

**Comment by jkbullard** [comment]
All those "installer" instances are probably interfering with each other. Uninstall Tunnelblick completely, including restarting your computer, and try to install again but only once If it fails, DO NOT install again. Look to see if the following file exists:

/Library/LaunchDaemons/net.tunnelblick.tunnelblick.tunnelblickd.plist

**Comment by gaivoronsky** [comment]
I've reinstalled OS X so now everything works perfectly. Thanks for the help

**Comment by gyosko** [comment]
I'm having this exact same issue on macOS Sierra 10.12.5.
Is there any way to solve this without reinstalling OS X? 

I did try to follow the instructions on superuser about the tmp folder, but nothing changed afterwards..

**Comment by jkbullard** [comment]
@gyosko - You're getting the same console messages about /tmp? If not, those instructions would not have helped. "Installation failed" can have many causes.

**Comment by gyosko** [comment]
@jkbullard - This is the only output I can get :

    2017-07-20 18:58:52.868 Tunnelblick[1050:31441] Beginning installation or repair
    2017-07-20 18:58:53.280 Tunnelblick[1050:31441] Need to replace and/or reload 'tunnelblickd':
    tunnelblickdHashOK   = YES
    launchctlPlistHashOK = YES
    tunnelblickdPlistOK  = YES
    socketOK             = NO
    2017-07-20 18:58:53.280 Tunnelblick[1050:31441] installer did not make the necessary changes

**Comment by ttben** [comment]
Same issue here
```
Tunnelblick installer started 2017-09-06 08:26:42. 1 arguments: 0x0017
Copied /Volumes/Tunnelblick/Tunnelblick.app to /Applications/Tunnelblick.app
Removed all 'com.apple.quarantine' extended attributes
Changed ownership of /Applications/Tunnelblick.app and its contents from 501:80 to 0:0
Need to replace and/or reload 'tunnelblickd':
    tunnelblickdHashOK   = YES
    launchctlPlistHashOK = YES
    tunnelblickdPlistOK  = YES
    socketOK             = NO
Replaced /Library/LaunchDaemons/net.tunnelblick.tunnelblick.tunnelblickd.plist
Used launchctl to load tunnelblickd
Tunnelblick installer finished without error
```

**Comment by colinbr96** [comment]
I was able to fix this issue for myself after nothing else seemed to work. It was my fault that I named my configuration file starting with a dot: `.config.ovpn` After renaming it, Tunnelblink was able to add it.

**Comment by jkbullard** [comment]
@colinbr96 - Thanks for your report.

However, you describe a very different problem than the others - they were installing Tunnelblick, not configurations. Also, the problem you had was not that the installation failed, but that the configuration is not available in Tunnelblick. Tunnelblick should warn about that.

For the record, **the original problem -- installing Tunnelblick** -- may have been fixed by c6b8e4cbb5223ed24872baf6d768e7b5721156c0, which allows installation of Tunnelblick even if the Tunnelblick daemon has been disabled, either manually or by some (unknown) "security" software.
